pages/1.11/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -10,70 +10,95 @@ enterprise: true
 
-To use this procedure, you must obtain the custom non-native Marathon tarball from your sales representative (<sales@mesosphere.io>). This custom tarball contains Marathon plus a secrets and auth plugin. These additional plugins allow you to use secrets and fine-grained access control in your non-native Marathon folder hierarchy.
+For this procedure, we are assuming that you have obtained an enterprise version of Marathon from your sales representative (<sales@mesosphere.io>). This custom tarball contains Marathon plus secrets and auth plugins. These additional plugins allow you to use secrets and fine-grained access control in your non-native Marathon folder hierarchy.
 
 **Prerequisites:**
 
 -  DC/OS and DC/OS CLI [installed](/1.11/installing/).
 -  [DC/OS Enterprise CLI 0.4.14 or later](/1.11/cli/enterprise-cli/#ent-cli-install).
--  Custom non-native Marathon tarball. Contact your sales representative or <sales@mesosphere.io> for access to this file.
 -  A private Docker registry that each private DC/OS agent can access over the network. You can follow [these](/1.11/deploying-services/private-docker-registry/) instructions for how to set up in Marathon, or use another option such as [DockerHub](https://hub.docker.com/), [Amazon EC2 Container Registry](https://aws.amazon.com/ecr/), and [Quay](https://quay.io/)).
+-  Custom non-native Marathon (version 0.11.0 or newer) image [deployed in your private Docker registry](/1.11/deploying-services/private-docker-registry#tarball-instructions). Contact your sales representative or <sales@mesosphere.io> for obtain the enterprise Marathon image file.
 -  You must be logged in as a superuser.
 -  SSH access to the cluster.
 
-# Step 1 - Load and Push the Custom Non-Native Marathon Image
-In this step, the custom non-native Marathon instance is pushed to the private Docker registry.
+<a name="variables-in-example"></a>
+**Variables in this example**
 
-1. Load the tarball into Docker with the custom non-native Marathon file (`marathon-dcos-ee.<version>.tar`) specified.
+Throughout this page, you will be instructed to invoke commands or take actions that differ according to your setup. For the parts that differ, we are using the `${VARIABLE}` notation, that should be replaced with the appropriate value that fits in your case.
 
-   ```bash
-   docker load -i marathon-dcos-ee.<version>.tar
-   ```
+The following table contains all the variables used in this page:
 
-    **Tip:** You can view the Marathon image with this command.
+| Variable | Description |
+|--------------------------|--------------------------------------------|
+| `${MY_USER_ID}` | The username that you use to login to your cluster. That's the username that you were prompted during `dcos cluster setup`. |
```

Comment:
s/to your cluster/into your cluster/ ?

Reply:
I think it should be "log in to your cluster" instead of "login to your cluster"

Reply:
✅